src/main/java/com/jcabi/dynamo/mock/MkTable.java

```diff
@@ -100,4 +100,16 @@ public Region region() {
     public String name() {
         return this.self;
     }
+
+    /**
+     * Delete item from aws table.
+     * @todo Implement #25
+     * @param attributes Attributes
+     * @throws IOException In case of DynamoDB failure
+     */
```

Comment:
I've added delete method to Table interface, implemented AwsTable.delete and a unit test for it. There are two another classes implementing Table: MkTable and ReTable. I've added these puzzles and submitted two tickets https://github.com/jcabi/jcabi-dynamo/issues/25 https://github.com/jcabi/jcabi-dynamo/issues/26 I understand that's how build&break principle should work, shouldn't it?